src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/OxoGReadCounts.java

```diff
@@ -81,6 +84,155 @@ public void annotate(final ReferenceContext refContext,
+                            if (insertionLength == pileupElement.getLengthOfImmediatelyFollowingIndel()) {
+                                final String insertionBases = pileupElement.getBasesOfImmediatelyFollowingInsertion();
+                                // edge case: ignore a deletion immediately preceding an insertion as p.getBasesOfImmediatelyFollowingInsertion() returns null [EB]
+                                if (insertionBases != null) {
+                                    final boolean isMatch = Allele.extend(referenceAllele, insertionBases.getBytes()).basesMatch(altAllele);
+                                    if (isMatch) {
+                                        pileupAllele = altAllele;
+                                    }
+                                }
+                            }
+                        }
+
+                        if (pileupElement.isBeforeDeletionStart()){
+                            final int deletionLength = pileupElement.getLengthOfImmediatelyFollowingIndel();
+                            if ((referenceAllele.getBases().length - altAllele.getBases().length) == deletionLength) {
+                                pileupAllele = altAllele;
+                            }
+                        }
+                        break;
+                    case MNP:
+                    case SNP:
+                        if (altAllele.basesMatch(getBasesForAlleleInRead(pileupElement, altAllele))) {
+                            pileupAllele = altAllele;
+                        }
+                        break;
+                    case MIXED:
+                    case SYMBOLIC:
+                    case NO_VARIATION:
+                    default:
+                        return;
+                }
+            }
+
+        } else {
+            pileupAllele = referenceAllele;
+        }
+
+        if (pileupAllele == null) {
+            return;
+        }
+
+        if (getMeanBaseQualityForAlleleInRead(pileupElement, pileupAllele) < meanBaseQualityCutoff) {
+            return;
+        }
+
+        if (countMap.containsKey(pileupAllele)) {
+            countMap.get(pileupAllele).increment();
+        }
+    }
+
+    private static byte[] getBasesForAlleleInRead(final PileupElement pileupElement, final Allele allele) {
+        final byte[] alleleBases = allele.getBases();
+        return ArrayUtils.subarray(pileupElement.getRead().getBases(), pileupElement.getOffset(), pileupElement.getOffset() + alleleBases.length);
+    }
+
+    private static double getMeanBaseQualityForAlleleInRead(final PileupElement pileupElement, final Allele allele) {
+        final byte[] alleleBases = allele.getBases();
+        final byte[] pileupBaseQualities = ArrayUtils.subarray(pileupElement.getRead().getBaseQualities(), pileupElement.getOffset(), pileupElement.getOffset() + alleleBases.length);
+
+        int totalBaseQuality = 0;
```

Comment:
`final int totalBaseQuality = MathUtils.sum(pileupBaseQualities);`

Reply:
Used min instead of mean.

No action.